updatehub/src/states/entry_point.rs

```diff
@@ -53,43 +59,71 @@ impl StateChangeImpl for State<EntryPoint> {
 
 create_state_step!(EntryPoint => Park);
 create_state_step!(EntryPoint => Poll);
+create_state_step!(EntryPoint => Probe);
 
-#[actix_rt::test]
-async fn polling_disable() {
+#[cfg(test)]
+mod tests {
```

Comment:
I am not against the test rework but it'd be nice if it could be done in a previous commit so the rework is isolated from this change.